### ISSUE TITLE
The recycling dashboard links to labels vs packaging

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/hunger-games#456]
### What
The recycling dashboard links to labels vs packaging
https://hunger.openfoodfacts.org/logos/deep-search?type=label&value=en:10-NiCD
https://hunger.openfoodfacts.org/logos/deep-search?type=packaging&value=en:01-PET

**Comment by alexfauquette** [comment]
How to solve:
in `LogoDefinition` as a `type` property which can be either 

```ts
type InsightType = 'label' | 'product_weight' | 'category' | 'expiration_date' | 'packager_code' | 'brand' | 'packaging' | 'qr_code'

export type LogoDefinition = {
  ...
  type: InsightType;
}
```

In `DashboardCard.ts` update `questionsUrl` and `logoAnnotationUrl` to use this new type property

In `LogoQuestionValidator.jsx` this new property will be available in `selectedOption` and should be used in the definition of `filterState`

**Comment by psankhe28** [comment]
I tried solving this issue by taking the help of the above-mentioned solution but after implementing steps 1 and 2, I am encountering the following error:
![image](https://user-images.githubusercontent.com/84843461/210585417-6c65d7fa-3563-4fbf-b18d-233218e6355d.png)

**Comment by psankhe28** [comment]
Okay. Thanks and sorry for the inconvenience.

**Comment by alexfauquette** [comment]
Do you want to open the PR and continue the issue?

**Comment by psankhe28** [comment]
Yeah!

**Comment by alexfauquette** [comment]
Fixed by #457
